spk/nzbhydra/Makefile

```diff
@@ -0,0 +1,48 @@
+SPK_NAME = nzbhydra
+SPK_VERS = 0.0.1a78
```

Comment:
This isn't correct, according to the `git clone` you do on R48. Either do a git checkout to a specific commit, or change this to use date (see similar packages).